scss/01-config/02-functions/_index.scss

```diff
@@ -13,17 +13,31 @@
 }
 
 @function em($px) {
-  @return #{math.div($px, $emBase)}em;
+  @return #{math.div(removeUnit($px), $emBase)}em;
 }
 
 @function rem($px) {
-  @return #{math.div($px, $emBase)}rem;
+  @return #{math.div(removeUnit($px), $emBase)}rem;
 }
 
+// when would we ever use this?
 @function px($em) {
```

Comment:
@dbox this is for converting em values to pixels. I've used it occasionally, doesn't hurt to have

Reply:
have mostly used it for debugging historically, not really something used in production

Reply:
cool would love to see a use case at some point. np!